packages/@pollyjs/adapter-node-http/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "@pollyjs/adapter-node-http",
+  "version": "1.3.2",
+  "description": "Node HTTP adapter for @pollyjs",
+  "main": "dist/cjs/pollyjs-adapter-node-http.js",
+  "module": "dist/es/pollyjs-adapter-node-http.js",
+  "files": [
+    "src",
+    "dist"
+  ],
+  "repository": "https://github.com/netflix/pollyjs/tree/master/packages/@pollyjs/adapter-node-http",
+  "license": "Apache-2.0",
+  "contributors": [
+    {
+      "name": "Yasin Uslu",
+      "email": "a.yasin.uslu@gmail.com"
+    },
+    {
+      "name": "Jason Mitchell",
+      "email": "jason.mitchell.w@gmail.com"
+    },
+    {
+      "name": "Offir Golan",
+      "email": "offirgolan@gmail.com"
+    }
+  ],
+  "keywords": [
+    "polly",
+    "pollyjs",
+    "record",
+    "replay",
+    "http",
+    "adapter"
+  ],
+  "publishConfig": {
```

Comment:
Should this be here?

Reply:
Yeah, thats how lerna knows that the package is public and should be published.

Reply:
TIL.  Sounds similar to pkg.json `"private"`

Reply:
Yeah, I'm not sure why they decided to have their own config for this...